src/templates/bitcoin.conf

```diff
@@ -4,7 +4,9 @@ regtest=1
 
 # Debugging
 debug=1
+debuglogfile=0
```

Comment:
This makes sense but breaks warcli debug_log. I think that's ok if maybe we have a new log category of commands that can use the same docker.api.log trick just target individual tanks

Reply:
ah yes, thank for thinking of that. If only we had a CI or something that could alert me of such cases.... :P

I think I'll push an additional commit using `docker.api.log` as you suggest

Reply:
Ok now done this in [a77ac80](https://github.com/bitcoin-dev-project/warnet/pull/133/commits/a77ac80af62dc732284e5b6caf108abe6f1f9536)